### ISSUE TITLE
I want to download redhat.vscode-community-server-connector-0.25.7.vsix

### ISSUE THREAD

**Issue by enjoicoder** [redhat-developer/rsp-server-community#231]
Since my development environment is a closed network, I need to download the vsix file and set it inside the vscode, and since the Java version uses 1.8, I need to use community-server-connector-0.25.7 to get 0.25.7 version for the vsix file, what should I do?

**Comment by rufus20145** [comment]
You can do it from within Visual Studio Code. Right-click on the extension and select "Install specific version".

**Comment by enjoicoder** [comment]
no,,I want to download to vsix file... it' just download extension files. 
I try to replacement file but it's not working..

**Comment by robstryker** [comment]
You can try this link?   https://marketplace.visualstudio.com/_apis/public/gallery/publishers/redhat/vsextensions/vscode-community-server-connector/0.25.7/vspackage

**Comment by enjoicoder** [comment]
I solved the problem by deleting the plugin, reinstalling the lower
version, and accessing that folder directly.

I saw your email late, but thank you.

2024년 12월 7일 (토) 오전 12:40, Rob Stryker ***@***.***>님이 작성:
